### PR TITLE
Image captcha

### PR DESCRIPTION
**Pull request by Wolfgang-Falk**
https://docs.lucee.org/reference/functions/imagecaptcha.html

Per Lucee Documentation for versions 5.x, I added the imagecaptcha() function page in /data/en and in /guides/en/lucee-only.

### CODE REVIEW

File: data/en/imagecaptcha.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	"type": "function",
+	"syntax": "imageCaptcha()",
+	"returns": "any",
+	"related": ["imagewritetobrowser"],
+	"description": "Provides information about the time when the current application scope was created.",
+	"params": [
+            {"name":"text","description":"Text in the image captcha.","values":[],"default":"","type":"string","required":true},
+            {"name":"height","description":"Height of the image captcha.","values":[],"default":"","type":"numeric","required":true},
+            {"name":"width","description":"Width of the image captcha.","values":[],"default":"","type":"numeric","required":true},
+            {"name":"difficulty","description":"Width of the image captcha.","values":[],"default":"medium","type":"string","required":false},
```
Comment (KamasamaK) [inline data/en/imagecaptcha.json:12]: According to the Lucee doc, the default is `low`. Also, the values should be `low`, `medium`, `high`.

File: data/en/imagecaptcha.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            {"name":"fonts","description":"One or more fonts for the CAPTCHA text, as an array or as string list (separated by commas). Lucee supports the system fonts recognized by JDK only.","values":[],"default":"arial","type":"any","required":false},
+            {"name":"fontsize","description":"Font size for text of the image captcha.","values":[],"default":"24","type":"any","required":false},
+            {"name":"fontcolor","description":"Font color for text of the image captcha.","values":[],"default":"black","type":"string","required":false}
+  ],
+	"engines": {
+		"lucee": {"minimum_version": "5", "notes": "", "docs": "https://docs.lucee.org/reference/functions/imagecaptcha.html"}
```
Comment (KamasamaK) [inline data/en/imagecaptcha.json:18]: It looks like the minimum version might be 5.3 based on some searching. Can anyone verify?

File: data/en/imagecaptcha.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+	"returns": "any",
+	"related": ["imagewritetobrowser"],
+	"description": "Provides information about the time when the current application scope was created.",
+	"params": [
+            {"name":"text","description":"Text in the image captcha.","values":[],"default":"","type":"string","required":true},
+            {"name":"height","description":"Height of the image captcha.","values":[],"default":"","type":"numeric","required":true},
+            {"name":"width","description":"Width of the image captcha.","values":[],"default":"","type":"numeric","required":true},
+            {"name":"difficulty","description":"Width of the image captcha.","values":[],"default":"medium","type":"string","required":false},
+            {"name":"fonts","description":"One or more fonts for the CAPTCHA text, as an array or as string list (separated by commas). Lucee supports the system fonts recognized by JDK only.","values":[],"default":"arial","type":"any","required":false},
+            {"name":"fontsize","description":"Font size for text of the image captcha.","values":[],"default":"24","type":"any","required":false},
```
Comment (KamasamaK) [inline data/en/imagecaptcha.json:14]: Type should be `numeric`.

File: data/en/imagecaptcha.json (hunk 1 above)
Comment (Wolfgang-Falk) [reply data/en/imagecaptcha.json:12]: Updated default to low.

File: data/en/imagecaptcha.json (hunk 3 above)
Comment (Wolfgang-Falk) [reply data/en/imagecaptcha.json:14]: Updated type to be numeric.

**Comment by pfreitag** [comment]
Looks like this is getting a JSON validation error `error:  /tmp/cfdocs/imagecaptcha.json: Unexpected token [ in JSON at position 132`

File: data/en/imagecaptcha.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
+{
+	"name": "imageCaptcha",
+	"type": "function",
+	"syntax": "imageCaptcha()",
+	"returns": "any",
+	"related": ["imagecreatecaptcha"], ["imagewritetobrowser"],
```
Comment (KamasamaK) [inline data/en/imagecaptcha.json:6]: This needs to be only one array

File: data/en/imagecaptcha.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+	"description": "Provides information about the time when the current application scope was created.",
+	"params": [
+            {"name":"text","description":"Text in the image captcha.","values":[],"default":"","type":"string","required":true},
+            {"name":"height","description":"Height of the image in pixels.","values":[""],"default":"","type":"numeric","required":true},
+            {"name":"width","description":"Width of the image in pixels.","values":[""],"default":"","type":"numeric","required":true},
```
Comment (KamasamaK) [inline data/en/imagecaptcha.json:11]: The `values` arrays here should be empty

**Review by KamasamaK — CHANGES_REQUESTED** [review]

File: data/en/imagecaptcha.json (hunk 5 above)
Comment (Wolfgang-Falk) [reply data/en/imagecaptcha.json]: Thanks, error corrected.

File: data/en/imagecaptcha.json (hunk 4 above)
Comment (Wolfgang-Falk) [reply data/en/imagecaptcha.json]: Thanks, error corrected.

**Comment by Wolfgang-Falk** [comment]
I looked further into the tests. Those are some neat tools. Since this is my first time adding a function page, I'll remember to update both /data/en/index.json and /data/en/functions.json with the newly added function in future branches.

Thank you for your assistance.

**Pull request merged**